jsapp/js/components/processing/sidebar/sidebarDisplaySettings.tsx

```diff
@@ -132,6 +222,20 @@ export default function SidebarDisplaySettings() {
               }
             })}
           </ul>
+
+          {props.asset?.survey &&
+            selectedDisplays.includes(StaticDisplays.Data) && (
+              <div className={styles.questionList}>
+                <strong>{t('Select the submission data to display.')}</strong>
+                <div className={styles.checkbox}>
+                  <MultiCheckbox
+                    type='bare'
+                    items={getCheckboxes()}
```

Comment:
There is a bug. Steps are:
1. Have a project with multiple questions and have multiple submissions submitted
2. In Display Settings change displayed questions (e.g. leave only one displayed)
3. See that in sidebar the questions are hidden
4. Change submission (header arrow up or down)
5. See that in sidebar the questions are hidden
6. In Display Settings you will notice that all questions are selected :bug:

Reply:
I think though that this is a semi good thing. Ideally we would like to keep all the display settings between the submissions, so the selected questions too.

So for some reason the selected questions are being reset

Reply:
I also see that these changes from our Zulip chat are not here yet:

- Put ~~it~~ *questions list* underneath "Submission data" - because if I understand this correctly, if I toggle off "Submission Data", all these checkboxes should be disabled? We could indent it from the left so it's clear it's a "sub section"
- Overflow the list of checkboxes; for max height I think we should go for 4.5, 5.5 or 6.5 (or more? not sure how much space we have there) of the height of single checkbox row (so If checkbox row is 24px, and we go with 5.5, max height would be 132px)
- After putting checkboxes underneath "Submission data" I think we don't need header inside the frame, but we could put some explanation above it (in regular text), like so (we already have "Select questions to be exported" in Data Downloads, so I think we can use something similar):
```
[x] SubmissionData
    Select questions to be displayed
    <checkboxes…>
```

Are you sure you've pushed all commits? :)

Reply:
> I think though that this is a semi good thing. Ideally we would like to keep all the display settings between the submissions, so the selected questions too.
> 
> So for some reason the selected questions are being reset

This actually should be fixed now by implementing what you said underneath (https://github.com/kobotoolbox/kpi/pull/4889#discussion_r1557802317)